src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -44,56 +38,6 @@ class FacilityDetail extends Component {
     document.title = this.__previousDocTitle;
   }
 
-  renderFacilityInfo() {
```

Comment:
Ugh yes, love it when we move from all these odd render functions into real function components. So much better and more stable.